library/swift/src/RequestTrailersBuilder.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-/// Builder used for constructing instances of `RequestTrailers` types.
+/// Builder used for constructing instances of `RequestTrailers` type.
```

Comment:
```suggestion
/// Builder used for constructing instances of the `RequestTrailers` type.
```

Reply:
This was "types" before since we have multiple subclasses, but no strong opinion (let's just keep them all consistent)